daemon/volumes.go

```diff
@@ -385,15 +385,14 @@ func copyExistingContents(source, destination string) error {
 // copyOwnership copies the permissions and uid:gid of the source file
 // into the destination file
 func copyOwnership(source, destination string) error {
-	var stat syscall.Stat_t
-
-	if err := syscall.Stat(source, &stat); err != nil {
+	stat, err := system.GetStat(source)
```

Comment:
Well, I first indeed used Lstat and no tests where failing, but I thought that using the original Stat would be less "risky", because I don't know why Stat was used instead of Lstat.